components/atom/table/src/index.scss

```diff
@@ -1,24 +1,33 @@
 @import '~@s-ui/theme/lib/index';
 
+$bgc-atom-table-header-cell: transparent !default;
+$bgc-atom-table-row: transparent !default;
+$bdrs-atom-table: $bdrs-none !default;
 $bgc-atom-table-row-actionable--hover: $c-gray-light-4 !default;
 $bgc-atom-table-row-actionable--active: $c-gray-light-3 !default;
 
 .react-AtomTable {
   border-collapse: collapse;
+  border-radius: $bdrs-atom-table;
   color: $c-gray-dark;
   font-size: $fz-s;
   padding: 0;
+  overflow: hidden;
 
   &--fullWidth {
     width: 100%;
   }
 
   &-headerCell {
+    background-color: $bgc-atom-table-header-cell;
     font-weight: $fw-regular;
     text-align: left;
   }
 
   &-row {
+    &:nth-child(2n + 2) {
```

Comment:
2n + 2 is the same as 2n

Reply:
i dk what does this rule purposes but i think its too much concrete for becoming generalist

Reply:
hmm, I think it's ok, it does what this PR is intending to do